### PR TITLE
Session reminder emails: mention editing responses #8708

### LINKED ISSUE #8708: Session reminder emails: mention editing responses
**Description of feature/enhancement**

Reminder emails sent to respondents have the following phrase:
>To submit your feedback for the above session, please go to this Web address ...

Suggestion: rephrase as
>To submit your feedback for the above session (or to edit/view responses submitted earlier), please go to this Web address ...

**Justification**
Some students contact us with requests such as 'I submitted already. Can you check if it was saved?'. The revised phrasing might reduce such support requests.

### PR DESCRIPTION
**Pull request by Vaggelis-Malandrakis**
Fixes #8708

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,5 +40,8 @@
     "lint:spaces": "lintspaces -n -t -d spaces -l 1 -. \"src/**/*.jsp\" \"src/**/*.tag\" \"src/main/**/*.html\" \"src/**/*.xml\" \"src/**/*.json\" \"src/**/*.css\" \"src/**/*.java\" \"src/main/webapp/dev/**/*.js\" \"src/**/*.properties\" \"*.js\" \"*.yml\" \"*.gradle\" \"static-analysis/*.*ml\"",
     "lint": "npm run lint:js && npm run lint:json && npm run lint:css && npm run lint:spaces"
   },
-  "private": true
+  "private": true,
+  "dependencies": {
+    "npm": "^5.8.0"
```
Comment (wkurniawan07) [inline package.json]: Can undo the changes here?

File: src/main/java/teammates/logic/api/EmailGenerator.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
             + "but you will not be able to edit existing responses or submit new responses";
 
     // feedback action strings
-    private static final String FEEDBACK_ACTION_SUBMIT = "submit";
+    private static final String FEEDBACK_ACTION_SUBMIT = "submit (or to edit/view)";
```
Comment (wkurniawan07) [inline src/main/java/teammates/logic/api/EmailGenerator.java]: - Note that by changing the string, the variable name `FEEDBACK_ACTION_SUBMIT` is no longer accurate.
- Phrase it as `submit, edit, or view` instead

Comment (Vaggelis-Malandrakis) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: Done!

File: package.json (hunk 1 above)
Comment (Vaggelis-Malandrakis) [reply package.json]: Sorry. Didn't notice that. It was automatically updated.

File: src/main/java/teammates/logic/api/EmailGenerator.java (hunk 2 above)
Comment (wkurniawan07) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: @Vaggelis-Malandrakis the second point is not addressed yet.

Comment (Vaggelis-Malandrakis) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: @wkurniawan07 What do you mean is not addressed? Sorry if I make silly questions. I am new to programming. I changed the variable's name everywhere in the class (from FEEDBACK_ACTION_SUMBIT to FEEDBACK_ACTION_SUBMIT_EDIT_OR_VIEW) as you suggested

Comment (wkurniawan07) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: > Phrase it as `submit, edit, or view` instead

The second point. I was referring to the words, not the variable name, there.

Comment (Vaggelis-Malandrakis) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: @wkurniawan07 I think now it's done!

Comment (Vaggelis-Malandrakis) [reply src/main/java/teammates/logic/api/EmailGenerator.java]: I have to run again the tests using God Mode so that's why the Travis check doesn't work
